sonar-scanner-engine/src/main/java/org/sonar/scanner/bootstrap/ScannerExtensionDictionnary.java

```diff
@@ -151,9 +150,11 @@ private static void completeBatchExtensions(ComponentContainer container, List<O
       }
       completePhaseDependencies(dag, extension);
     }
-    List sortedList = dag.sort();
+    List<?> sortedList = dag.sort();
 
-    return Collections2.filter(sortedList, Predicates.in(extensions));
+    return (Collection<T>) sortedList.stream()
+      .filter(o -> extensions.contains(o))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1612)